helper/users/lookup_test.go

```diff
@@ -39,3 +43,37 @@ func TestLookup_NobodyIDs(t *testing.T) {
 	must.Eq(t, 65534, uid) // ubuntu
 	must.Eq(t, 65534, gid) // ubuntu
 }
+
+func TestWriteFileFor_Linux(t *testing.T) {
+	// This is really how you have to retrieve umask. See `man 2 umask`
+	umask := syscall.Umask(0)
```

Comment:
`syscall` is (mostly?) deprecated in favor of `unix`

https://pkg.go.dev/golang.org/x/sys/unix#Umask

Reply:
Can we just deprecate `umask` instead? (kidding kidding fix pushed but syscall import remains due to use of `Stat_t`)